### ISSUE TITLE
A11y_Code.visualstudio.com_Visual studio code_Top navigation bar_Reflow: In Reflow, we are getting both horizontal and vertical scroll bars in the page.

### ISSUE THREAD

**Issue by MukallaDivya** [microsoft/vscode-docs#5156]
## GitHub Tags:
#A11yTCS;#Win10;#WCAG1.4.10;#DesktopWeb;#A11ySev2;#ChromiumEdge;#P1-VisualStudioCodeServices-Web-Feb2022;#P1_Visual Studio Code Services;#Visual Studio Code Services;
## Environmental Details:
#URL: https://code.visualstudio.com/License
## OS & Browser Details :
Windows 10 Enterprise 21H2(OS Build 19044,1466)
Microsoft Edge Version 98.0.1108.43 (Official build) (64-bit)
## Pre-Requisite:
To apply reflow to the page, follow the below steps:
1. Set display resolution to : - 1280 * 1024.
2. Increase your browser Zoom level to 400%.
## Repro Steps :
1. Launch the URL https://code.visualstudio.com/License.
2. apply Reflow for the page.
3. Check whether the content is properly visible or not.
## Actual Result :
when we apply reflow to the page , we are getting scroll bar In both directions "horizontal and vertical" .
## Expected Result :
there should be scroll bar in one direction only , either in horizontal or vertical.
## User Impact:
Users may got confuse , while navigating with scroll bars in two directions.
## Attachments:

https://user-images.githubusercontent.com/99255594/153322277-276fa064-016d-49f2-bdc2-9131312dd4e8.mp4
![Bug_5156_Reflow_both directional scroll bars](https://user-images.githubusercontent.com/99255594/153393354-2cb84d3b-b125-47e3-a159-cf4338c5272a.jpeg)

**Comment by rzhao271** [comment]
Ref https://github.com/microsoft/vscode-website/commit/5f0cab27d1efa81de6bec248e0e56152ff43ef7b.
I accidentally referenced the wrong issue in the commit message.

**Comment by MukallaDivya** [comment]
Check again this bug in the below Environment.
URL : https://code.visualstudio.com/License
         https://code.visualstudio.com/blog
Now, The  scroll bar is in one direction only, that is in vertical direction.
so, the issue is fixed.
![Screenshot (128)](https://user-images.githubusercontent.com/99255594/157234926-aed7901b-060d-4467-92d9-86d3110d5048.png)
